pkg/agent/flowexporter/exporter/exporter_test.go

```diff
@@ -131,17 +135,70 @@ func getElementList(isIPv6 bool) []ipfixentities.InfoElementWithValue {
 		antreaIE = AntreaInfoElementsIPv6
 	}
 	for _, ie := range ianaIE {
-		elemList = append(elemList, ipfixentities.NewInfoElementWithValue(ipfixentities.NewInfoElement(ie, 0, 0, ipfixregistry.IANAEnterpriseID, 0), nil))
+		element, _ := registry.GetInfoElement(ie, ipfixregistry.IANAEnterpriseID)
+		ieWithValue, _ := ipfixentities.DecodeAndCreateInfoElementWithValue(element, nil)
```

Comment:
A similar utility like the one in flow_aggregator_test.go can be used here.

Reply:
I see similar code when sending templates in flow exporter and flow aggregator. We should probably add an utility in go-ipfix and use it there.